app/client/lib/appStart.js

```diff
@@ -4,9 +4,12 @@ if(location.hostname !== 'localhost' && location.hostname !== '127.0.0.1')
 
 
 // Make sure the example contract code is up to date
-var shortExample = Helpers.defaultContractExample.substr(Helpers.defaultContractExample.indexOf("\n\n")+2);
-if (localStorage.getItem('contractSource') && localStorage.getItem('contractSource').indexOf(shortExample) !== -1) {
-    localStorage.setItem('contractSource', Helpers.defaultContractExample);
+var shortExample = Helpers.getDefaultContractExample().substr(Helpers.getDefaultContractExample().indexOf("\n\n")+2);
```

Comment:
This could be better done, by passing `true` to the  `Helpers.getDefaultContractExample` functioning simply returning just the text without pragma.
Then we don't need to cut strings but simply return a new custom one.